src/sagemaker/workflow/pipeline.py

```diff
@@ -824,15 +828,34 @@ def list_steps(self):
         """Describes a pipeline execution's steps.
 
         Returns:
-             Information about the steps of the pipeline execution. See
+             Information about the steps of the pipeline execution in a Pandas table. See
              `boto3 client list_pipeline_execution_steps
              <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/\
 sagemaker.html#SageMaker.Client.list_pipeline_execution_steps>`_.
         """
+        try:
+            import pandas as pd
```

Comment:
pandas is currently supported by the python sdk, see feature_group.py

We're safe to import it directly at the top of the file